fcq/templates/fcq/fcq_display.html

```diff
@@ -0,0 +1,331 @@
+{% extends 'base.html' %}
+
+
+{% block content %}
+
+<br>
+<h1>Search for Professors</h1>
+<div id="error"></div>
+<div class="col-12 sticky-top" style="background-color: whitesmoke;">
+    <br>
+    <div class="width=100%row" id="search_div" align="right">
```

Comment:
the html attribute "align" is deprecated in HTML5. Change this to CSS with a style tag. I believe float right would have a similar effect

Reply:
somewhat low priority, but probably worth our time